src/styles/header.scss

```diff
@@ -0,0 +1,16 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    position: relative;
+    width: 40px;
+    height: 40px;
+    left: 50px;
```

Comment:
Please use some css properties order, for example it's better to group by type in this case:
    _position: relative;
    left: 50px;_
    width: 40px;
    height: 40px;

You can follow next articles and get more info https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/
https://developer.mozilla.org/en-US/docs/Web/CSS/font-family
https://9elements.com/css-rule-order/